ores/utilities/label_reverted.py

```diff
@@ -5,11 +5,11 @@
 <rev_id>\t<reverted>
 
 Usage:
-    features_reverted -h | --help
-    features_reverted --api=<url> [--revert-radius=<revs>]
-                                  [--revert-window=<secs>]
-                                  [--rev-pages=<path>]
-                                  [--rev-reverteds=<path>]
+    label_reverted -h | --help
+    label_reverted --api=<url> [--revert-radius=<revs>]
```

Comment:
I think we should drop those notebooks in a separate commit.  I don't think that they are really that useful for the project at this time.